hack/openshift_security.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+PROJECT="default"
+oc create sa kubevirt-controller
+oc adm policy add-cluster-role-to-user cluster-admin -z kubevirt-controller
+oc patch deployment/virt-controller --patch '{"spec":{"template":{"spec":{"serviceAccountName": "kubevirt-controller"}}}}'
+oc create sa kubevirt-iscsi
+oc adm policy add-scc-to-user hostmount-anyuid -z kubevirt-iscsi
```

Comment:
Is it really not possible to run the oc commands agains the service accounts created by our manifests? Then you don't have to patch all the deployments.

Reply:
sure, we can do that